scripts/release.sh

```diff
@@ -39,6 +52,9 @@ echo "[Minor version only] Update the dependencies to RA packages in the create-
 echo "Press Enter when this is done"
 read
 
+# Get the current version from package.json
+npm_current_package_version=$(jq -r '.version' ./packages/react-admin/package.json)
```

Comment:
suggestion: it's hard to tell the difference between `npm_current_package_version` and `npm_package_version`. Could you rename `npm_current_package_version` to `npm_previous_package_version`?